globus_cli/services/transfer/async_transfer.py

```diff
@@ -100,18 +102,27 @@
 @click.option('--recursive', is_flag=True,
               help=('source-path and dest-path are both directories, do a '
                     'recursive dir transfer. Ignored in batchmode'))
-@click.option('--sync-level', default="mtime", show_default=True,
+@click.option('--label', default=None, help=('Set a label for this task.'))
+@click.option('--sync-level', default=None, show_default=True,
               type=CaseInsensitiveChoice(
                   ("exists", "size", "mtime", "checksum")),
               help=('How will the Transfer Task determine whether or not to '
                     'actually transfer a file over the network?'))
+@click.option('--preserve-mtime', is_flag=True, default=False,
+              help=('Preserve file and directory modification times.'))
+@click.option('--verify-checksum/--no-verify-checksum', default=True,
+              show_default=True,
+              help=('Verify checksum after transfer.'))
+@click.option('--encrypt', is_flag=True, default=False,
```

Comment:
This is the only default that I'm not sure makes sense.
I assume it's the same as the default for the web app, but I'd sooner have a `--no-encrypt` to turn it off than `--encrypt` to turn it on. Non-blocker for merging, however.